jobbergate-cli/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "jobbergate-cli"
+version = "1.1.1"
+description = "Jobbergate CLI Client"
+authors = ["Omnivector Solutions <info@omnivector.solutions>"]
+license = "MIT"
+packages = [ { include = "jobbergate_cli" } ]
+
+
+[tool.poetry.dependencies]
+python = "^3.6.2"
+click = "^8.0.1"
+inquirer = "^2.7.0"
+PyJWT = "<2"
+PyYAML = "^5.4.1"
+python-dotenv = "^0.19.0"
+requests = "^2.26.0"
+tabulate = "^0.8.9"
+urllib3 = "^1.26.7"
+sentry-sdk = "^1.4.3"
+boto3 = "^1.18.64"
+loguru = "^0.5.3"
+
+[tool.poetry.dev-dependencies]
+black = "^21.9b0"
+pytest-cov = "^3.0.0"
+pytest-random-order = "^1.0.4"
+pytest-freezegun = "^0.4.2"
+pytest-responsemock = "^1.0.1"
+flake8 = "^3.9.2"
+isort = "^5.9.3"
+pytest-env = "^0.6.2"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--random-order --cov=jobbergate_cli --cov-report=term-missing --cov-fail-under=25"
```

Comment:
can you do this as list like [here](https://github.com/omnivector-solutions/armada-api/blob/main/pyproject.toml#L74-L82)? I think it's better to see

Reply:
I think you changed another pyproject.toml. This one remains the same

Reply:
Oh, I changed in jobbergate-api. I'll update it in jobbergate-cli as well.